specs/fault-proof-system-v2/improvements.md

```diff
@@ -0,0 +1,57 @@
+# Improvements
+
+<!-- All glossary references in this file. -->
+
+[g-checkpoint-output]: ../glossary.md#checkpoint-output
+[g-validator]: ../glossary.md#validator
+[g-l2-output]: ../glossary.md#l2-output-root
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Guaranteed Win of Valid Output](#guaranteed-win-of-valid-output)
+- [Guaranteed Data Verifiability for ZK Proof Verification](#guaranteed-data-verifiability-for-zk-proof-verification)
+- [Robust to Delay Attacks](#robust-to-delay-attacks)
+- [Accurate Identification of First Disagreeing Block](#accurate-identification-of-first-disagreeing-block)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+Fault Proof System V2 has been designed to address the key limitations identified in
+[Fault Proof System V1](../fault-proof/challenge.md), as outlined in the [Background](./README.md#background). This
```

Comment:
The link is broken because the README.md file is being converted to index.html. What if we rename it to a different filename that matches the title?